workspaces/src/network/mainnet.rs

```diff
@@ -0,0 +1,90 @@
+use std::path::PathBuf;
+use std::str::FromStr;
+
+use async_trait::async_trait;
+use url::Url;
+
+use near_primitives::views::FinalExecutionStatus;
+
+use crate::network::Info;
+use crate::network::{
+    Account, CallExecution, CallExecutionDetails, NetworkClient, NetworkInfo,
+    TopLevelAccountCreator,
+};
+use crate::rpc::{client::Client, tool};
+use crate::types::{AccountId, InMemorySigner, SecretKey};
+use crate::Contract;
+
+const RPC_URL: &str = "https://rpc.mainnet.near.org";
+const HELPER_URL: &str = "https://helper.mainnet.near.org";
```

Comment:
Do we have a helper on the mainnet? Who is paying for that?

Reply:
Aww man, didn't realize that mainnet doesn't have a helper contract to aid account creation. What's the flow look like for doing that for mainnet instead?

Reply:
you have to call the `near` contract directly to create a "sub account" on it.

Here's a snippet from nearapijs:
```
await creatorAccount.functionCall(
      'near',
      'create_account',
      {
        new_account_id: process.argv[3],
        new_public_key: process.argv[4]
      },
      '300000000000000',
      utils.format.parseNearAmount('0.1')
    )
    ```

Reply:
yeah, actually since mainnet requires an `Account` to pay the transaction of creating an `Account`, `create_tla` just won't be supported for mainnet. Instead, we'll have to have an existing `Account` on mainnet and do `Account::from_file()` then we can do `account.create_subaccount` to actually be able to create accounts